docs/ingestion-beam/index.md

```diff
@@ -349,12 +349,12 @@ unmodified.
 #### GeoIP Lookup
 
 1. Extract `ip` from the `x_forwarded_for` attribute
-   * when the `x_pipeline_proxy` attribute is not present, use the
+    * when the `x_pipeline_proxy` attribute is not present, use the
      second-to-last value (since the last value is a forwarding rule IP
      added by Google load balancer)
-   * when the `x_pipeline_proxy` attribute is present, use the third-to-last
+    * when the `x_pipeline_proxy` attribute is present, use the third-to-last
      value (since the tee introduces an additional proxy IP)
-   * fall back to the `remote_addr` attribute, then to an empty string
+    * fall back to the `remote_addr` attribute, then to an empty string
```

Comment:
Bullet points were not indented [on the doc page](https://mozilla.github.io/gcp-ingestion/ingestion-beam/#decoder-job)